frontend/packages/knative-plugin/locales/ja/knative-plugin.json

```diff
@@ -0,0 +1,142 @@
+  "Loading Bootstrap Servers...": "ブートストラップサーバーを読み込んでいます...",
+  "No Topics found": "トピックが見つかりません",
+  "Add Topics": "トピックの追加",
+  "{{kafkaTopicErrorMessage}}. Try adding Topics manually.": "{{kafkaTopicErrorMessage}}。トピックの手動による追加を試行してください。",
+  "Loading Topics...": "トピックを読み込んでいます...",
+  "Bootstrap Servers": "ブートストラップサーバー",
+  "The address of the Kafka broker": "Kafka ブローカーのアドレス",
+  "Topics": "トピック",
+  "Virtual groups across Kafka brokers": "Kafka ブローカー全体の仮想グループ",
+  "Consumer Group": "コンシューマーグループ",
+  "A group that tracks maximum offset consumed": "消費された最大オフセットを追跡するグループ",
+  "Subject": "サブジェクト",
+  "apiVersion": "apiVersion",
+  "Kind": "種類",
+  "Match Labels": "ラベルに一致する",
+  "Add Values": "値の追加",
+  "A Universal Resource Indicator where events are going to be delivered. Ex. \"http://cluster.example.com/svc\"": "イベントが配信されるユニバーサルリソースインジケーター (例: \"http://cluster.example.com/svc\")",
+  "Enter URI": "URI の入力",
+  "This resource will be the Sink for the Event Source.": "このリソースはイベントソースのシンクになります。",
+  "No resources available": "利用可能なリソースがありません",
+  "Select the URI option, or exit this form and create a Knative Service, Broker, or Channel first.": "URI オプションを選択するか、またはこのフォームを終了して、Knative Service、Broker、または Channelを最初に作成します。",
+  "Select resource": "リソースの選択",
+  "Sink": "シンク",
+  "Add a Sink to route this Event Source to a Channel, Broker, Knative Service or another route.": "シンクを追加して、このイベントソースを Channel、Broker、Knative Service または別のルートにルーティングします。",
+  "Channel": "チャネル",
+  "Create a Knative Channel to create an event forwarding and persistence layer with in-memory and reliable implementations": "Knative Channel を作成し、インメモリーの信頼性の高い実装を備えたイベント転送および永続化層を作成します",
+  "Please enter a URI that is less then 2000 characters.": "2000 文字未満の URI を入力してください。",
+  "Invalid URI.": "URI が無効です。",
+  "Required": "必須",
+  "Cannot be longer than 253 characters.": "253 文字を超えることはできません。",
+  "No model registered for {{referenceForKnEventSource}}": "{{referenceForKnEventSource}} のモデルは登録されていません",
+  "Event Source cannot be created": "イベントソースを作成できません",
+  "Creation of event sources are not currently supported on this cluster.": "イベントソースの作成は、このクラスターでは現在サポートされていません。",
+  "Event Sources": "イベントソース",
+  "Create an event source to register interest in a class of events from a particular system": "特定のシステムから興味のあるイベント分野を登録するためにイベントソースを作成します",
+  "Service Account Name": "サービスアカウント名",
+  "Select a Service Account Name": "サービスアカウント名の選択",
+  "The name of Service Account use to run this": "これを実行するために使用するサービスアカウントの名前",
+  "Configurations": "設定",
+  "No Configurations found for this resource.": "このリソースの設定が見つかりません。",
+  "Latest Created Revision name:": "作成された最新のリビジョン名:",
+  "Latest Ready Revision name:": "準備状態の最新のリビジョン名:",
+  "Deployment": "デプロイメント",
+  "No Deployment found for this resource.": "このリソースのデプロイメントが見つかりません。",
+  "Hide filters": "フィルターの非表示",
+  "Show filters": "フィルターの表示",
+  "No {{title}} found for this resource.": "このリソースの {{title}} が見つかりません。",
+  "Broker": "Broker",
```

Comment:
If "Broker" can be translated in Japanese?

Reply:
This looks all set.